### ISSUE TITLE
Editor text colors not working

### ISSUE THREAD

**Issue by scooterx3** [damiankorcz/Prism-Theme#178]
**Describe the bug**
In the Style Settings plugin I have edited Editor > Editor Text Color > Dark Theme (and Light Theme) > Italic Text Color Preset (and Bold and Italic / Bold). But the actual color of the bold, italicized or both won't actually change. 

**To Reproduce**
- Go to settings 
- Style Settings
- Editor
- Editor Text Color
- Dark Theme
- Italic Text Color Preset - change it to something other than default 
- Go to any document and italicize some text and observe whether the color matches what you picked in settings or not

**Expected behavior**
I expect the color of the italicized text to match what was entered into settings

**Screenshots**
![image](https://github.com/user-attachments/assets/b1cb0c41-5022-4974-85de-8c596357d902)
![image](https://github.com/user-attachments/assets/05efb930-14a8-4cc1-9f62-98b3c37868a5)

**Setup Information**
 - Device Type: PC
 - Operating System: Ubuntu 22.04
 - Prism version: 3.5.3
 - Color Schemes Used: Raven
 - Obsidian Version: 1.6.7
 - Obsidian Installer Version: 1.6.7
 - Related Community plugins: I only have 'QuickAdd' and 'Style Settings' installed, nothing more. 

**Additional context**
n/a

**Comment by damiankorcz** [comment]
Hi @scooterx3,
I looking into this and can't replicate the issue. Can you make sure your Style Settings version is up to date? Should be on 1.0.9 at the moment.

If that doesn't work can you send me a copy of your Style Settings config? If you go to Style Settings you can click Export at the top and paste that for me. After that try resetting Prism style settings to default with the button next to the main dropdown.

Just for more info, what method did you use to install Obsidian?

**Comment by scooterx3** [comment]
Thanks @damiankorcz for taking the time to look at this. I went fiddling a bit. I realize that I usually pack around a .obsidian folder and put it into new vaults as I create them. The reason for that is so I can preserve (among possibly other things) the keyboard shortcuts that I like to use. It seems to have had some junk in it that clashed.

So I started a brand new vault and installed Prism and Style Settings and the coloring worked perfectly. 

I went and deleted some extra themes I had in .obsidian/themes called 'Blue Topaz' and 'Things'. Upon their removal and restart of Obsidian, the coloring worked perfectly. I've since reinstalled Blue Topaz and Things, but can no longer reproduce the issue.

I've got an edge case here that I don't think I can really get to the bottom of at this point, but at least the issue is solved :grin:
